docs/resources/machine_configuration_apply.md

```diff
@@ -73,11 +77,22 @@ Required:
 - `client_key` (String, Sensitive) The client key
 
 
+<a id="nestedatt--on_destroy"></a>
+### Nested Schema for `on_destroy`
+
+Optional:
+
+- `graceful` (Boolean) Graceful indicates whether node should leave etcd before the upgrade, it also enforces etcd checks before leaving. Default true
+- `reboot` (Boolean) Reboot indicates whether node should reboot or halt after resetting. Default false
+- `reset` (Boolean) Reset the machine to the initial state (STATE and EPHEMERAL will be wiped). Default false
```

Comment:
this is a bit confusing... if this is false, it doesn't reset?

this looks same level as `graceful`

Reply:
I mean it could be `enabled` or something like that?

Reply:
it's the default value the provider sets, the user can ovveride